### PR TITLE
Allow jobs to be manually triggered from cron jobs

### PR DESCRIPTION
**Pull request by EliZucker**
Hi all,

These changes address #3080. I added a trigger button next to cron jobs that manually starts them. My changes were based on a development last year that allowed this feature via the CLI: https://github.com/jingxu97/kubernetes/commit/5a304c021fcf213e087023654c941ac238d1b232

This is my first pull request on this project, so I would appreciate any feedback. Thanks!

### CODE REVIEW

File: src/app/backend/resource/cronjob/jobs.go [hunk 1]
```diff
@@ -144,3 +145,41 @@ func filterJobsByState(active bool, jobs []batch.Job) (matchingJobs []batch.Job)
...
+  annotations["cronjob.kubernetes.io/instantiate"] = "manual"
+
+  labels := make(map[string]string)
+  for k, v := range cronJob.Spec.JobTemplate.Labels {
+    labels[k] = v
+  }
+
+  jobToCreate := &batch.Job{
+    ObjectMeta: metaV1.ObjectMeta{
+      // job name cannot exceed DNS1053LabelMaxLength (52 characters)
```
Comment (rf232) [inline src/app/backend/resource/cronjob/jobs.go]: CronJob name can be long already though, and especially in a UI, we should perhaps help the enduser with that.

**Comment by EliZucker** [comment]
@austbot Thank you for the prompt feedback! I tried a small tweak to increase codecove, but I will do more tomorrow if that doesn't work.

@rf232 The triggered job's name is now automatically shortened if needed. Let me know if I need to explain/change anything else.

**Comment by EliZucker** [comment]
@rf232 Would you mind reviewing this PR again? I think your earlier concern is addressed and it should be mergeable. Thanks!

**Pull request merged**